Detectors/TRD/calibration/src/KrClusterFinder.cxx

```diff
@@ -29,7 +29,7 @@ double KrClusterFinder::LandauChi2Functor::operator()(const double* par) const
   // par[1] : location parameter (approximately most probable value)
   // par[2] : sigma
   double retVal = 0;
-  for (unsigned int i = xLowerBound; i <= xUpperBound; ++i) {
+  for (int i = xLowerBound; i <= xUpperBound; ++i) {
```

Comment:
Seems unrelated, but minor change.

Reply:
This belongs to the other commit which addresses some of the compiler warnings which are sent around via a weekly mail now.